README.md

```diff
@@ -89,39 +94,39 @@ A curated list of awesome R frameworks, packages and software. Inspired by [awes
 * [ggvis](https://github.com/rstudio/ggvis) - Interactive grammar of graphics for R.
 * [Leaflet](http://rstudio.github.io/leaflet/) - One of the most popular JavaScript libraries interactive maps.
 * [MetricsGraphics](http://hrbrmstr.github.io/metricsgraphics/) - Enables easy creation of D3 scatterplots, line charts, and histograms.
-* [networkD3](http://christophergandrud.github.io/networkD3/) - D3 JavaScript Network Graphs from R.
+* [networkD3](http://christophergandrud.github.io/networkD3/) and [ndtv](https://cran.r-project.org/web/packages/ndtv/) - D3 JavaScript Network Graphs from R.

```

Comment:
`ndtv` tends to be more specific than all other packages in that section. I have moved it to the "Network Analysis" section, just after `networkDynamic`, which it complements, but please of course feel free to move it to "Graphic Displays".